README.md

```diff
@@ -35,6 +45,14 @@ npm run larva
 
 This will open a development server at localhost:3000. You can contribute to patterns in the packages/larva-patterns, and see the updates reflected here.
 
+### Changing the Default Port
+
+To modify the port Larva runs on, you can use `NODE_PORT` before your `npm run larva` command,
+
+```bash
+ NODE_PORT=8080 npm run larva
```

Comment:
Do we want everything to use the `npx @penske-media-corp/larva start` command for starting the server? Or is it useful to retain `npm run larva`? 

Related - are there any updates needed to the scripts in package.json, and are there any notes needed in UPGRADING.md regarding steps themes can take when they update to this version?

Reply:
Until we've made these decisions, I'm considering the `npx` usage an optional approach (even if only for a week or two while we complete these efforts). My intent is to get these changes deployed, and then revisit documentation appropriately.

I can definitely make a note in the UPGRADING docs, especially since this will be our 1.0.0 release.

Using an example project's `package.json`, I believe we can consolidate from ~20 scripts, to this,

```
"scripts": {
	"backstop": "backstop --config=node_modules/@penskemediacorp/backstopjs-config",
	"larva": "npx @penskemediacorp/larva start",
	"prod": "npx @penskemediacorp/larva prod",
	"dev": "npx @penskemediacorp/larva dev",
	"lint": "npx @penskemediacorp/larva lint",
	"upgrade": "npx @penskemediacorp/larva upgrade"
}
```

This should avoid any confusion or breaking changes from documentation, make managing these scripts across a projects substantially easier, and helps maintain a sane developer experience.

Or we could take it a step further,
```
"scripts": {
	"backstop": "backstop --config=node_modules/@penskemediacorp/backstopjs-config",
	"larva": "npx @penskemediacorp/larva",
}
```

However developers would need to remember the list of commands (although `--help` would alleviate that). I think it's worth bringing to the group for some additional consideration.

Thoughts?

Reply:
Makes sense! This is a very satisfying improvement :-D